src/core/analysis/mcmc/PathSampler.cpp

```diff
@@ -62,7 +62,7 @@ double PathSampler::marginalLikelihood( void ) const
     
         // now we can compute the marginal likelihood
         // the method uses the trapezoidal rule for numerical integration
-        for (size_t i = 0; i < pathValues.size()-1; ++i)
+        for (size_t i = 0; i+1 < pathValues.size(); ++i)
```

Comment:
This was a fix for a bug that I ran into.  If `pathValues.size()` is 0, then `pathValues.size()-1` is 4294967296, because `pathValues.size()` is unsigned.